test/dash/dash_parser_manifest_unit.js

```diff
@@ -1551,9 +1551,12 @@ describe('DashParser Manifest', () => {
     expect(stream).toBeUndefined();
   });
 
-  it('override manifest value if ignoreMinBufferTime is true', async () => {
+  /** @type {shaka.extern.ManifestConfiguration} */
+  /** @return {!Promise.<shaka.extern.Manifest>} */
+  function getParsedManifest(config) {
```

Comment:
I think you made a mistake in your annotations.  To annotate a parameter, use the `@param` tag, and put that and `@return` into the same jsdoc comment:

```js
  /**
   * @param {shaka.extern.ManifestConfiguration} config
   * @return {!Promise.<shaka.extern.Manifest>}
   */
  function getParsedManifest(config) {
```